/**:
  ros__parameters:
    # MPC params
    mode: "faultfree"
    # mode: "dummy"
    horizon: 10
    time_step: 0.1
    trajectory_tracking: true
    param_set: "P1"
    solver_opts:
      ipopt.tol: 1e-5
    # Trajectory params
    traj_shape: "hover"
    # traj_shape: "generate_point_stabilizing"
    traj_duration: 30
    tuning:
      P1: 
        Q: [10.0, 10.0, 10.0, 1.0, 1.0, 0.1]
        # R: [0.1, 0.1, 0.1]
        R: [1.0, 1.0, 1.0]
        R_full: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
        P_mult: 1.0  # P = P_mult * Q
      P2: 
        Q: [5.0, 5.0, 10.0, 0.2, 0.2, 0.1]
        R: [1.0, 1.0, 1.0]
        R_full: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
        P_mult: 1.0  # P = P_mult * Q
